### ISSUE TITLE
Test file not loading during MACE training - shows 0 energy/forces while train file loads correctly

### ISSUE THREAD

**Issue by Long-Brian-Yang** [ACEsuit/mace#1060]
**Describe the bug**
During MACE fine-tuning, the test file is not being loaded correctly. The training file loads successfully with complete energy and forces data, but the test file shows 0 energy and 0 forces in the logs, despite both files having identical format and structure.

**Expected behavior**
INFO: Test file 1/1 [1400 configs, 1400 energy, 1400 forces, 1400 stresses] loaded from 'data/test.xyz'
INFO: Total test set (1400 configs):
INFO: Default_default: 1400 configs, 1400 energy, 1400 forces, 1400 stresses

**Actual (Current issue):**
INFO: Test set 1/1 [energy: 0, stress: 0, virials: 0, dipole components: 0, head: 1080, forces: 0, charges: 0]
INFO: Total Test set [energy: 0, stress: 0, virials: 0, dipole components: 0, head: 1080, forces: 0, charges: 0]

**Environment:**
- MACE version: 0.3.13

**Comment by ilyes319** [comment]
Could you install and test it on the latest develop branch 0.3.14.

**Comment by Long-Brian-Yang** [comment]
Thanks! The issue is resolved now. The test file loads correctly.
